src/patterns/molecules/error/error.scss

```diff
@@ -0,0 +1,45 @@
+@use "../../../shared-styles/normalize"; // Always used first
+@use "../../../shared-styles/baseline";
+@use "../../../shared-styles/grid";
+@use "../../../shared-styles/media-query";
+@use "../../../shared-styles/rem";
+@use "../../../shared-styles/scale";
+@use "../../../shared-styles/size";
+@use "../../../shared-styles/space";
+@use "../../../shared-styles/typography";
+
+.error {
+  text-align: center;
+  @include size.max-inline-size(grid.$max-inline-size);
+  @include space.margin(auto, inline);
+  @include space.padding(grid.$column_gap, inline);
+  @include space.block($start: rem.calculate(baseline.for-units(12)), $end: rem.calculate(baseline.for-units(12)));
+  @include media-query.mq($from: medium) {
+    @include space.block($start: rem.calculate(baseline.for-units(20)), $end: rem.calculate(baseline.for-units(20)));
+  }
+
+  @supports (display: flex) {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  > :last-child {
+    @include space.margin(0, block-end);
+  }
+
+}
+
+.error__image {
+  @include space.block($end: baseline.for-units(4));
+
+  @supports (display: flex) {
+    display: block;
+  }
+}
+
+.error__heading {
+  @include space.block($end: rem.calculate(baseline.for-units(2)));
```

Comment:
Possibly but I always think having mixins within mixins could confuse and how many mixins we want to nest before it's too much?

Reply:
My thinking is one well chosen name could make the code easier to read, rather than having to mentally parse the nested call in the style rules on each occurrence 🤔